app/page/template/partials/template-message.htm

```diff
@@ -134,7 +134,7 @@
         <file-asset class="message-asset" data-bind="css: {'ephemeral-asset-expired icon-file': message.isObfuscated()}" params="message: message"></file-asset>
       <!-- /ko -->
       <!-- ko if: asset.is_location() -->
-        <location-asset params="asset: asset"></location-asset>
+        <location-asset params="asset: asset, locationRepository: $parents[1].locationRepository"></location-asset>
```

Comment:
`$parents[1].locationRepository` looks fragile to me. Can we pass in the location repository somehow else?

Reply:
Yeah it's definitely aweful. Unfortunately I don't see much options in there (not saying that they don't exist). That's how viewModels are given to the view.
If someone is aware of a better solution, I am all ears